ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -165,14 +177,14 @@ private[dao] final class TransactionsReader(
       startExclusive: Offset,
       endInclusive: Offset,
       requestingParties: Set[Party],
-      verbose: Boolean,
+      eventProjectionProperties: EventProjectionProperties,
   )(implicit
       loggingContext: LoggingContext
   ): Source[(Offset, GetTransactionTreesResponse), NotUsed] = {
     val span =
       Telemetry.Transactions.createSpan(startExclusive, endInclusive)(qualifiedNameOfCurrentFunc)
     logger.debug(
-      s"getTransactionTrees($startExclusive, $endInclusive, $requestingParties, $verbose)"
+      s"getTransactionTrees($startExclusive, $endInclusive, $requestingParties, $eventProjectionProperties)"
```

Comment:
I am not sure what to do here with the logging... what is important then later to be shown?

Reply:
Let's preserve the pattern of logging the arguments.